<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForecastWindow</class>
 <widget class="QDialog" name="ForecastWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rates and Forecasts</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>711</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="RateForecastPlotWidget" name="rate_forecast_plot" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>711</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="modelSelectorComboBox">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>10</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="VoxelViewWidget" name="voxel_plot" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>341</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="log_likelihood_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Likelihood</string>
   </property>
  </widget>
  <widget class="QLabel" name="fc_time_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Forecast time</string>
   </property>
  </widget>
  <widget class="QLabel" name="pred_rate_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Predictated total rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Forecast Results</string>
   </property>
  </widget>
  <widget class="QLabel" name="rating_time_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Model Performance</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RateForecastPlotWidget</class>
   <extends>QWidget</extends>
   <header>plots</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VoxelViewWidget</class>
   <extends>QWidget</extends>
   <header>plots</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
